src/test/java/org/jenkinsci/utils/process/ProcessUtilsTest.java

```diff
@@ -0,0 +1,22 @@
+package org.jenkinsci.utils.process;
+
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+
+import org.junit.Test;
+
+public class ProcessUtilsTest {
+
+    @Test
+    public void testToString() throws IOException, InterruptedException {
+        ProcessBuilder pb = new ProcessBuilder("java");
+        pb.inheritIO();
```

Comment:
This causes all the output from `java` to come to stdout/err, which fills up the log when doing `mvn verify`.